src/components/reservation/reservation-modal.tsx

```diff
@@ -50,26 +90,81 @@ const ReserVationModal = ({
       screenReaderModalDescriptionText={t("screenReaderModalDescriptionText")}
       closeModalAriaLabelText={t("ariaLabelModalTwoText")}
     >
-      <div>
-        <div>
-          {" "}
-          <Cover pid={pid as Pid} size="xlarge" animate />
-        </div>
-        <div>
-          {" "}
-          <Button
-            label={t("reserveText")}
-            buttonType="none"
-            variant="filled"
-            disabled={false}
-            collapsible={false}
-            size="large"
-            onClick={onClick}
-          />
+      <section className="reservation">
+        <header className="reservation__header">
+          <Cover pid={pid as Pid} size="medium" animate />
+          <div className="reservation__header__container">
+            <div className="reservation__header__tag">
+              {materialTypes[0].specific}
+            </div>
+            <h2 className="reservation__header__title">{titles.main[0]}</h2>
+            <p className="reservation__header__author">
+              {t("materialHeaderAuthorByText")} {author} (
+              {publicationYear.display})
+            </p>
+          </div>
+        </header>
+        <div className="reservation__form">
+          <div className="reservation__form__submit">
+            <p className="reservation__form__info">
+              {`${t("weHaveShoppedText")} ${totalMaterials} ${t(
+                "copiesThereIsText"
+              )} ${totalReservations} ${t("reservationsForThisMaterialText")}`}
+            </p>
+            <Button
+              label={t("approveReservationText")}
+              buttonType="none"
+              variant="filled"
+              disabled={false}
+              collapsible={false}
+              size="small"
+              onClick={onClick}
+            />
+          </div>
+          <div className="reservation__form__list">
+            {edition?.summary && (
+              <ReservationFormListItem
+                icon={Various}
+                title={t("editionText")}
+                text={edition.summary}
+              />
+            )}
+            {userData?.patron?.defaultInterestPeriod && (
+              <ReservationFormListItem
+                icon={LoanHistory}
+                title={t("haveNoInterestAfterText")}
+                text={getNoInterestAfter(userData.patron.defaultInterestPeriod)}
+              />
+            )}
+            {userData?.patron?.preferredPickupBranch && branchData && (
```

Comment:
I think we should destruct properties from user data as we do with manifestation